pkg/assembler/backends/neo4j/pkg.go

```diff
@@ -204,10 +195,6 @@ func (e *namespaceToName) Type() string {
 	return "PkgHasName"
 }
 
-func (e *namespaceToName) Nodes() (v, u assembler.GuacNode) {
```

Comment:
same for all these in the neo4j backend

Reply:
Removed a bunch of stuff, PTAL

Reply:
https://github.com/guacsec/guac/pull/1035/commits/181ff6a0cd4f7250a07dc8319b61f7e843983993